translations/ar/LC_MESSAGES/messages.po

```diff
@@ -181,19 +226,19 @@ msgstr ""
 
 #: /content/includes/home.yaml
 msgid "bbc_news"
-msgstr ""
+msgstr "ال بي بي سي"
 
 #: /content/includes/home.yaml
 msgid "buzzfeed"
-msgstr ""
+msgstr "باز فيد"
 
 #: /content/includes/home.yaml
 msgid "condenast"
 msgstr ""
 
 #: /content/includes/home.yaml
 msgid "daily_mail"
-msgstr ""
+msgstr "البريد اليوميه"
```

Comment:
I changed the first suggestion but I don't think this one should be changed. I used the female form for "Daily" in Arabic because it's not the mail that's daily it's the publication and that is female so the word daily should follow that.